boilerplate/openshift/golang-osd-operator/rvmo-bundle.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-REPOSITORY=${REPOSITORY:-"git@github.com:openshift/managed-release-bundle-osd.git"}
+REPOSITORY=${REPOSITORY:-"https://github.com/openshift/managed-release-bundle-osd.git"}
```

Comment:
why?

Reply:
We use a github app to auth with a token. To my knowledge we can't use ssh.